ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/OnDisk.hs

```diff
@@ -83,15 +81,9 @@ type NextBlock blk = LDB.NextItem blk
   Deprecated functions
 -------------------------------------------------------------------------------}
 
-{-# DEPRECATED initLedgerDB "Use Ouroboros.Consensus.Storage.LedgerDB (initLedgerDB)" #-}
+{-# DEPRECATED initLedgerDB "initLedgerDB will throw an error if used. Use Ouroboros.Consensus.Storage.LedgerDB (initialize)" #-}
```

Comment:
If this will throw an error, is it ok to simply deprecate it? Why not remove it altogether?